plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/messages.properties

```diff
@@ -1,12 +1,12 @@
+BUTTON_ACCOUNTS_PANEL_ADD_ACCOUNT=Add Account
+BUTTON_ACCOUNTS_PANEL_LOGOUT=Sign Out
```

Comment:
You might want to try using `SWT.FLAT` in your buttons.  They're smaller and look a bit better in the dialog IMHO.